src/components/FiltersDrawer/index.tsx

```diff
@@ -130,18 +131,58 @@ const FiltersDrawer: React.FC<FiltersDrawerProps> = ({
                 <Overline>
                   Row {e[0]} to {e[1]}
                 </Overline>
-              </ExpandedLineWrapper>
+              </NonFilterElement>
             ))
           ) : (
-            <ExpandedLineWrapper data-cy="no-expanded-lines-message">
+            <SectionWrapper data-cy="no-expanded-lines-message">
               <Body>No lines have been expanded.</Body>
-            </ExpandedLineWrapper>
+            </SectionWrapper>
+          )}
+        </StyledSideNavGroup>
+
+        <StyledSideNavGroup
```

Comment:
We discussed this offline but i agree it would be really nice to refactor this out. The LG SideNavGroup component is being pretty difficult so i am going to defer this to a follow up ticket.